tidb-cloud/integrate-tidbcloud-with-airbyte.md

```diff
@@ -0,0 +1,99 @@
+
+![TiDB source configuration](/media/tidb-cloud/integration-airbyte-parameters.jpg)
+
+You can use any combination of sources and destinations, such as TiDB to Snowflake, and CSV files to TiDB.
+
+For more details about the TiDB connector, see [TiDB Source](https://docs.airbyte.com/integrations/sources/tidb) and [TiDB Destination](https://docs.airbyte.com/integrations/destinations/tidb).
+
+## Set up the connection
+
+After setting up the source and destination, you can build and configure the connection.
+
+The following steps use TiDB as both source and destination. Other connectors may have different parameters.
+
+1. Click **Connections** in the sidebar and then click **New Connection**.
+2. Select the previously established source and destination.
+3. Go to the **Set up** connection panel and create a name for the connection, such as `${source_name} - ${destination-name}`.
+4. Set **Replication frequency** to **Every 24 hours**, which means the connection replicates data once a day.
+5. Set **Destination Namespace** to **Custom format** and set **Namespace Custom Format** to **test** to store all data in the `test` database.
+6. Choose the **Sync mode** to **Full refresh | Overwrite**.
+
+    > **Tip:**
+    >
+    > The TiDB connector supports both Incremental and Full Refresh syncs.
+    >
+    > - In Incremental mode, Airbyte only reads records added to the source since the last sync job. The first sync using Incremental mode is equivalent to Full Refresh mode.
+    > - In Full Refresh mode, Airbyte reads all records in the source and replicates to the destination in every sync task. You can set the sync mode for every table named **Namespace** in Airbyte individually.
+
+    ![Set up connection](/media/tidb-cloud/integration-airbyte-connection.jpg)
+
+7. Set **Normalization & Transformation** to **Normalized tabular data** to use the default normalization mode, or you can set the dbt file for your job. For more information about normalization, refer to [Transformations and Normalization](https://docs.airbyte.com/operator-guides/transformation-and-normalization/transformations-with-dbt).
+8. Click **Set up connection**.
+9. Once the connection is established, click **ENABLED** to activate the synchronization task. You can also click **Sync now** to sync immediately.
+
+![Sync data](/media/tidb-cloud/integration-airbyte-sync.jpg)
+
+## Limitations
+
+- The TiDB connector does not support the Change Data Capture (CDC) feature.
+- TiDB destination converts the `timestamp` type to the `varchar` type in default normalization mode. It happens because Airbyte converts the timestamp type to string during transmission, and TiDB does not support `cast ('2020-07-28 14:50:15+1:00' as timestamp)`.
+- For some large ETL missions, you need to increase the parameters of [transaction restrictions](/develop/dev-guide-transaction-restraints.md#large-transaction-restrictions) in TiDB.
```

Comment:
```suggestion
- For some large ELT missions, you need to increase the parameters of [transaction restrictions](/develop/dev-guide-transaction-restraints.md#large-transaction-restrictions) in TiDB.
```
Please confirm whether it's ETL or ELT in this case. As far as i know, they are alternatives.

Reply:
We use ETL https://docs.pingcap.com/tidb/dev/overview#use-cases

Reply:
Airbyte introduces itself as ELT, so we should use ELT here.